docs/src/getting-started/updating-linuxcnc.adoc

```diff
@@ -20,7 +20,7 @@ updates. If you don't have an internet connection to your PC see
 == Upgrade to the new version
 
 This section describes how to upgrade LinuxCNC from version 2.8 to the
-new 2.9.1 version.  It assumes that you have an existing 2.8 install that you
+new 2.9.2 version.  It assumes that you have an existing 2.8 install that you
```

Comment:
Maybe this should be 2.9.x ?

Reply:
Done, made it 2.9.y to avoid the impression that 2.8.x and 2.9.x would be the same x. I know ... just some gut feeling that we have a special audience.